vendor/noise_handshake/src/handshake.rs

```diff
@@ -114,36 +113,33 @@ impl<IO: AsyncRead + AsyncWrite> Future for NoiseHandshake<IO> {
                     },
                 ..
             } => {
-                let msg = format!(
+                trace!(
```

Comment:
:x: I think instead of trace, we actually don't want any logging at all here. Once this works, nobody is going to look at it again.